fuzz/fuzz_targets/roundtrip-wit.rs

```diff
@@ -462,7 +502,6 @@ mod generate {
                 ret.push_str("\n\n");
             }
 
-            self.types_in_interface.clear();
```

Comment:
Whoops, sorry for not catching that either!

Reply:
Heh I'm the one who ran the fuzzer for hours without noticing this 🤡